functions/helpers.ts

```diff
@@ -6,7 +6,9 @@ import { getGiftCardById } from "./post-order";
 import { fallbackIntlMastercard, fallbackIntlVisa, masterCardIntlSkus, visaIntlSkus } from "./reloadly-lists";
 import { AccessToken, ReloadlyFailureResponse } from "./types";
 
-export const allowedChainIds = [1, 5, 100, 31337];
+export const permitAllowedChainIds = [1, 5, 100, 31337];
```

Comment:
There's I think 12 networks permit2 is deployed on. Might want to check their docs or blockscan

Reply:
We can add all but it doesn't make sense to do it because we only generate permits on Gnosis

Reply:
I guess if it's a big project we can handle in another pull but my understanding is that we have support for any network with permit2 @rndquu rfc

Reply:
Yes we can generate permits on any network that has permit2 contract deployed.

Reply:
I only found this in their blog: 
Permit2 is deployed to the [same address](https://etherscan.io/address/0x000000000022D473030F116dDEE9F6B43aC78BA3) across mainnet, Ethereum, Optimism, Arbitrum, Polygon, and Celo.
But it seems outdated because it's deployed on Gnosis